benchmarks/dynamo/common.py

```diff
@@ -298,11 +298,13 @@ def timed(
         xm.mark_step()
         xm.wait_device_ops()
 
-    t0 = time.perf_counter()
+    time_total = 0
     # Dont collect outputs to correctly measure timing
     for _ in range(times):
         # Put this call inside the loop to reset the seed for each iteration.
```

Comment:
Hi @shunting314 , why do we have to move `reset_rng_state` for `PyTorch/XLA` in the first place?

Reply:
Hey @desertfire , it's because we want to reset rng state for every iteration we run inside the loop. let me know if you still want more context